`dkms build` looks for 'armv7l' folder instead of 'arm' folder

I don't know if this is a dkms problem or the configuration problem, but when I run:

```
sudo dkms install -m $DRV_NAME -v $DRV_VERSION
```

I get the error:

```
'make' KVER=4.4.38-v7+....(bad exit status: 2)
ERROR (dkms apport): binary package for rtl8812AU: 4.3.20 not found
Error! Bad return status for module build on kernel: 4.4.38-v7+ (armv7l)
Consult /var/lib/dkms/rtl8812AU/4.3.20/build/make.log for more information.
```

Unfortunately I've successfully built it since then so I don't have the errors from the 'make.log'. But the solution for me was what I saw in [this Stack Overflow post](https://stackoverflow.com/questions/39775844/how-to-make-dkms-pass-the-right-arch-to-make), which was to make a symlink to the arm folder:

```
sudo ln -s /usr/src/linux-headers-4.4.38-v7+/arch/arm/ /usr/src/linux-headers-4.4.38-v7+/arch/armv7l
```

So, once you know about it the fix is relatively easy.

make.log with the latest Raspbian 9.4 kernel, in my case for ```dkms build -m $DRV_NAME -v $DRV_VERSION```
```
DKMS make.log for rtl8812AU-4.3.20 for kernel 4.14.52-v7+ (armv7l)
Tue Jul 10 03:28:28 CEST 2018
make ARCH=armv7l CROSS_COMPILE= -C /lib/modules/4.14.52-v7+/build M=/var/lib/dkms/rtl8812AU/4.3.20/build  modules
make[1]: Entering directory '/usr/src/linux-headers-4.14.52-v7+'
Makefile:638: arch/armv7l/Makefile: No such file or directory
make[1]: *** No rule to make target 'arch/armv7l/Makefile'.  Stop.
make[1]: Leaving directory '/usr/src/linux-headers-4.14.52-v7+'
Makefile:1687: recipe for target 'modules' failed
make: *** [modules] Error 2
```

--- a/README.md
+++ b/README.md
@@ -15,6 +15,21 @@ The driver (versions) are primarily targetted at [Debian](https://www.debian.org
 
 I have currently successfully compiled the driver on Debian kernel 3.2, 3.16, 4.1-4.15 and on Raspbian with kernel 3.18, 4.1 and 4.4.
 
+## Raspberry Pi
+There is a section added for compilation on the Raspberry Pi, but it is not enabled by default.
+To enable it, make the following change in the `Makefile`:
+```
+CONFIG_PLATFORM_I386_PC = y
+CONFIG_PLATFORM_ARM_RPI = n
+```
+to
+```
+CONFIG_PLATFORM_I386_PC = n
+CONFIG_PLATFORM_ARM_RPI = y
+```
+And then it should compile succesfully **on** the Raspberry Pi.
+Keep in mind that when using dkms on the Raspberry Pi you need to make the above change before running the dkms installation commands.
+
 ## DKMS
 [DKMS](http://linux.dell.com/dkms/) is a system which will automatically recompile and install a kernel module when a new kernel gets installed or updated.
 To make use of DKMS, install the `dkms` package, which on Debian (based) systems is done like this:

